hack/verify-golint.sh

```diff
@@ -25,7 +25,11 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 if ! which golint > /dev/null; then
   echo 'Can not find golint, install with:'
   echo 'go get -u golang.org/x/lint/golint'
-  exit 1
+  go get -u golang.org/x/lint/golint
```

Comment:
is it a normal pattern to just install automatically as part of the build? I prefer to be in control of what gets installed on my machine!

Reply:
ok, remove it